RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift

```diff
@@ -28,16 +21,31 @@ extension AccountPortfoliosClient: DependencyKey {
 
 		/// Update when hidden resources change
 		Task {
+			@Dependency(\.resourcesVisibilityClient) var resourcesVisibilityClient
 			for try await hiddenResources in await resourcesVisibilityClient.hiddenValues().removeDuplicates() {
 				guard !Task.isCancelled else { return }
 				await state.updatePortfoliosHiddenResources(hiddenResources: hiddenResources)
 			}
 		}
+	}
+
+	public static let liveValue: AccountPortfoliosClient = {
+		let state = State()
+
+		@Dependency(\.onLedgerEntitiesClient) var onLedgerEntitiesClient
+		@Dependency(\.cacheClient) var cacheClient
+		@Dependency(\.tokenPricesClient) var tokenPricesClient
+		@Dependency(\.appPreferencesClient) var appPreferencesClient
+		@Dependency(\.resourcesVisibilityClient) var resourcesVisibilityClient
+		@Dependency(\.gatewaysClient) var gatewaysClient
+
+		Self.registerSubscribers(state: state)
```

Comment:
Is this change just a refactor for cleanup, or do we actually need to call the static method for subscriptions to take place? Because if we do, and it is a consequence of `SWIFT_STRICT_CONCURRENCY`, then there are other clients where we need to follow the path

Reply:
This is necessary to resolve the compiler error: `Default argument cannot be both actor-isolated and actor-isolated`. The issue arises from using `state` (which is an actor) inside a Task. If other clients follow this pattern, they will encounter the same error. I’m not entirely sure why calling the static method resolves the issue, but it works...

Reply:
thanks for the explanation. Guess that if it is compiling on other clients (where we do create `Task`s that handle suscriptions but probably without modifying an `actor`)  then we are fine